### ISSUE TITLE
[Usability]Missing Confirmation after adding Members to organization

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9609]
Copied from: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1853045/
Steps to reproduce:

Task 6: Add a member to your organization
Issue Details:
Participant was unsure if he had added a member and would like a confirmation message after Add
